code/modules/vueui/ui.dm

```diff
@@ -124,23 +124,23 @@ main ui datum.
   * @return html code - text
   */
 /datum/vueui/proc/generate_html(var/css_tag)
-#ifdef UIDEBUG
-	var/debugtxt = "<div id=\"dapp\"></div>"
-#else
 	var/debugtxt = ""
-#endif
+	if(user && check_rights(R_DEV, FALSE, user=user))
+		debugtxt = "<div id=\"dapp\"></div>"
```

Comment:
Note: UIDEBUG also makes in each open call resend assets in case if they changed instead of using cached assets in resource file.

Reply:
yeah, that's why i didn't remove the UIDEBUG define